src/components/details.js

```diff
@@ -0,0 +1,61 @@
+import { useEffect } from 'react';
```

Comment:
- [ ] The details page is beautiful, however:

![image](https://user-images.githubusercontent.com/57421823/152480181-feb9eaab-5fd3-4157-9f1f-ccdb6251f3cb.png)

There seems a problem with your header horizontally overflowing the page.  Kindly fix it.

That way, we meet this requirement:

![image](https://user-images.githubusercontent.com/57421823/152480289-8b50d40f-0c94-47eb-9510-b63ea9a3deca.png)

... And your app will look beautiful and striking!! 💪